### PR TITLE
✨ Added theme switch and remove default map theme

### PR DESCRIPTION
**Pull request by wjtje**
Theses changes are needed for https://github.com/EqualStreetNames/equalstreetnames/pull/203

### CODE REVIEW

File: html/en/index.html [hunk 1]
```diff
@@ -129,9 +135,9 @@ <h5 class="modal-title">
     <script src="../../../website/src/script/index.ts"></script>
     <script>
         app.countries = "";
-        app.style = "mapbox://styles/mapbox/dark-v10";
+        app.style = undefined;
```
Comment (jbelien) [inline html/en/index.html]: You can even remove the line !

```suggestion
```

**Review by jbelien — CHANGES_REQUESTED** [review]

File: html/en/index.html (hunk 1 above)
Comment (wjtje) [reply html/en/index.html]: Yes that's possible, but now it's clear that the map style can be changed.

Comment (jbelien) [reply html/en/index.html]: I'll add instructions in the `README.md` file.

**Review by jbelien — APPROVED** [review]
Looks good !
Waiting for https://github.com/EqualStreetNames/equalstreetnames/pull/203 to be merged.

**Pull request merged**